packages/terra-form-select/src/search/Frame.jsx

```diff
@@ -204,7 +204,7 @@ class Frame extends React.Component {
    * Handles the blur event.
    */
   handleBlur(event) {
-    const { relatedTarget } = event;
+    const relatedTarget = event.relatedTarget || document.activeElement;
```

Comment:
I think we need similar changes for tag and multiple variant as well : 

<img width="1680" alt="Screen Shot 2021-03-17 at 12 35 45 PM" src="https://user-images.githubusercontent.com/12869809/111428443-82037c80-871d-11eb-8d7a-630ccf6bdd1d.png">

Reply:
If we need more changes for the other variants, could you log a new issue to address them?